client-js/app/scripts/app.ts

```diff
@@ -247,17 +257,102 @@ window.addEventListener('WebComponentsReady', () => {
     }
   }
 
+  class DatavaultLiveActivity implements Activity {
```

Comment:
How come there's a DatavaultLiveActivity in addition to the DatavaultActivity? 

Plots of datasets opened by themselves, without navigating to ```?live``` update when new data is added live.  

What's the distinction for these separate activities?